dependencies/pom.xml

```diff
@@ -755,6 +756,23 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>com.oracle.oci.sdk</groupId>
+                <artifactId>oci-java-sdk-bom</artifactId>
```

Comment:
You need to move this at the bottom of the file. Dependency management cannot be overridden and import of BOM have surprises. [oci-java-sdk-bom](https://repo1.maven.org/maven2/com/oracle/oci/sdk/oci-java-sdk-bom/1.31.0/oci-java-sdk-bom-1.31.0.pom) has [oci-java-sdk](https://repo1.maven.org/maven2/com/oracle/oci/sdk/oci-java-sdk/1.31.0/oci-java-sdk-1.31.0.pom) as parent and it defines some non oci-sdk related dependency management that will conflict with Helidon. If fact we may want to avoid using the bom entirely because of that ; we need to check with @barchetta that the non oci-sdk related dependencies are acceptable.

Reply:
For now I have completely removed the bom. The dependancies we have now are sufficient. It looks better now.